pages/_document.tsx

```diff
@@ -38,7 +38,7 @@ export default function Document() {
         <meta name='theme-color' content='#671E75' />
         <link rel='manifest' href='/manifest.json' />
```

Comment:
Web app manifest should have the filename extension 'webmanifest'.

FURTHER READING
[Learn why this is important and how to fix it on webhint.io](https://webhint.io/docs/user-guide/hints/hint-manifest-file-extension/?source=devtools)

Reply:
Will do!